libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/ConwayFeatures.hs

```diff
@@ -371,15 +371,16 @@ vote pf govActionId =
 -- time that it takes for a DRep to expire become smaller
 -- =====================================================
 
--- | Test that the expiration of Proposals works.
---   We start with 1 proposal in the current state, and 0 proposals in the previous state (inside the DRepPulser)
---   The one current proposal is proposed in Epoch 0, and expires after Epoch 2. So the number of proposals in
---   each Epoch should look ike this
+-- | Test that the expiration of Proposals works. We start with 1 proposal in the current
+-- state, and 0 proposals in the previous state (inside the DRepPulser) The one current
+-- proposal is proposed in Epoch 0, and expires after Epoch 2. So the number of proposals
+-- in each Epoch should look like this:
+--
 --    Epoch   current  prev
 --    0       1        0
 --    1       1        1
 --    2       1        1
---    3       0        1
+--    3       1        1
```

Comment:
@TimSheard This is the part that looked wrong to me when we discussed the DRep pulser during one of the meetings a little over a week ago. Sure enough, after fixing the implementation the test failed right at this point. Let me know if you have questions why this is the case, I'll be happy to elaborate.

Reply:
It is a bit surprising that the initial current proposal in epoch0 , i set to expire after epoch 2, so why is is still in effect in epoch 3?

Reply:
> why is is still in effect in epoch 3

Because that governance action will be removed only after enactment, which is delayed by the whole epoch. Here is a ticket about it in #3804